File: ৫২টি প্রোগ্রামিং সমস্যা ও সমাধান/প্রোগ্রামিং সমস্যা ১ — জোড়-বিজোড় ১.py
T = int(input())  # Take the number of test cases

for i in range(1, T + 1):  # Loop from 1 to T (inclusive)
    n = int(input())  # Take input for each test case
    
    if n % 2 == 0:  # Check even/odd
        print("even")
    else:
        print("odd")

File: ৫২টি প্রোগ্রামিং সমস্যা ও সমাধান/প্রোগ্রামিং সমস্যা ১ — জোড়-বিজোড় ১.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        
        Scanner sc = new Scanner(System.in);
        
        int T = sc.nextInt();  // Read the number of test cases
        
        for (int i = 1; i <= T; i++) {
            int n = sc.nextInt();  // Read each number
            
            if (n % 2 == 0) {  
                System.out.println("even"); 
                
            } else {
                System.out.println("odd");
            }
        }
        
        sc.close();  // Close the scanner (best practice)
    }
}